src/main/scala/beam/sim/CustomTerminateAtFixedIterationNumber.scala

```diff
@@ -0,0 +1,24 @@
+package beam.sim
```

Comment:
Is this used? Using the Github UI I don't see any actual usage.

Reply:
Or is this used by virtue of extending TerminationCriterion - thus the way binding is done?

Reply:
yes, this is used in BeamHelper to rewrite behaviour implemented in matsim
[here](https://github.com/LBNL-UCB-STI/beam/pull/2151/files?file-filters%5B%5D=.conf&file-filters%5B%5D=.scala#diff-d4f57c2659d44575c5f00b19c7e0449cR163)